extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/pages/contract_agreements/services/ContractAgreementPageCardBuilder.java

```diff
@@ -45,11 +49,11 @@ public class ContractAgreementPageCardBuilder {
 
     @NotNull
     public ContractAgreementCard buildContractAgreementCard(
-            @NonNull ContractAgreement agreement,
-            @NonNull ContractNegotiation negotiation,
-            @NonNull Asset asset,
-            @NonNull List<TransferProcess> transferProcesses
-    ) {
+        @NonNull ContractAgreement agreement,
+        @NonNull ContractNegotiation negotiation,
+        @NonNull Asset asset,
+        @NonNull List<TransferProcess> transferProcesses,
+        @NonNull Map<String, SovityContractTerminationRecord> terminations) {
```

Comment:
style question on the brackets: which one is better for Git?

Reply:
okay, if it's like that, we can adjust the IJ code style via editorconfig to auto-format that, any time you say "split into lines" or format a multi line expression. we can do that.

Then we could that apply at once. Also a previous employer of mine had arrived at the same conclusion after some back and forth.

Reply:
(ofc not in this PR, but when we do that big "apply the IJ Editorconfig)